src/main/scala/org/broadinstitute/hail/driver/ExportVariantsSolr.scala

```diff
@@ -179,6 +185,24 @@ object ExportVariantsSolr extends Command with Serializable {
         cc
       }
 
+    //delete and re-create the collection
```

Comment:
Can you add a flag like `--drop` to control the delete/create block?  Otherwise you can't load multiple datasets into the same collection.